File: integration/connexion3/src/entities/menu.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.util.Date;

/**
 *
 * @author User
 */
public class menu {
    int id;
    String entre;
    String plat;
    String dessert;
    String date;

    public menu() {
    }
    

    public menu(int id, String entre, String plat, String dessert, String date) {
        this.id = id;
        this.entre = entre;
        this.plat = plat;
        this.dessert = dessert;
        this.date = date;
    }

    public menu(String entre, String plat, String dessert, String date) {
        this.entre = entre;
        this.plat = plat;
        this.dessert = dessert;
        this.date = date;
    }

    
   
    public menu(String entre, String plat, String dessert) {
        this.entre = entre;
        this.plat = plat;
        this.dessert = dessert;
    }

    public menu(int id, String entre, String plat, String dessert) {
        this.id=id;
         this.entre = entre;
        this.plat = plat;
        this.dessert = dessert;
     
    }

    public int getId() {
        return id;
    }

    public String getEntre() {
        return entre;
    }

    public String getPlat() {
        return plat;
    }

    public String getDessert() {
        return dessert;
    }

    public String getDate() {
        return date;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setEntre(String entre) {
        this.entre = entre;
    }

    public void setPlat(String plat) {
        this.plat = plat;
    }

    public void setDessert(String dessert) {
        this.dessert = dessert;
    }

    public void setDate(String date) {
        this.date = date;
    }

    @Override
    public String toString() {
        return "menu{" + "id=" + id + ", entre=" + entre + ", plat=" + plat + ", dessert=" + dessert + ", date=" + date + '}';
    }
    

    
    
}

File: evenements/src/eventBundle/Entity/menu.php
<?php

namespace eventBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * menu
 *
 * @ORM\Table(name="menu")
 * @ORM\Entity(repositoryClass="eventBundle\Repository\menuRepository")
 */
class menu
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="entre", type="string", length=255)
     */
    private $entre;

    /**
     * @var string
     *
     * @ORM\Column(name="plat", type="string", length=255)
     */
    private $plat;

    /**
     * @var string
     *
     * @ORM\Column(name="dessert", type="string", length=255)
     */
    private $dessert;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="datetime")
     */
    private $date;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set entre
     *
     * @param string $entre
     *
     * @return menu
     */
    public function setEntre($entre)
    {
        $this->entre = $entre;

        return $this;
    }

    /**
     * Get entre
     *
     * @return string
     */
    public function getEntre()
    {
        return $this->entre;
    }

    /**
     * Set plat
     *
     * @param string $plat
     *
     * @return menu
     */
    public function setPlat($plat)
    {
        $this->plat = $plat;

        return $this;
    }

    /**
     * Get plat
     *
     * @return string
     */
    public function getPlat()
    {
        return $this->plat;
    }

    /**
     * Set dessert
     *
     * @param string $dessert
     *
     * @return menu
     */
    public function setDessert($dessert)
    {
        $this->dessert = $dessert;

        return $this;
    }

    /**
     * Get dessert
     *
     * @return string
     */
    public function getDessert()
    {
        return $this->dessert;
    }

    /**
     * Set date
     *
     * @param \DateTime $date
     *
     * @return menu
     */
    public function setDate($date)
    {
        $this->date = $date;

        return $this;
    }

    /**
     * Get date
     *
     * @return \DateTime
     */
    public function getDate()
    {
        return $this->date;
    }
}

